.fvm/fvm_config.json

```diff
@@ -1,3 +1,3 @@
 {
-  "flutterSdkVersion": "3.10.6"
+  "flutterSdkVersion": "3.13.0"
```

Comment:
Is it necessary to upgrade the sdk version ? As long as we don't need new features or bugfixes I prefer keeping the version as is

Reply:
this version fixes some issues with material3, its is not an absolute necessity but as we still are in the development process with no production release we might as well follow the sdk versions, we don't really need stability currently